tools/accuracy_checker/accuracy_checker/evaluators/custom_evaluators/sequential_action_recognition_evaluator.py

```diff
@@ -413,10 +414,11 @@ def automatic_model_search(self, network_info):
             if len(model_list) > 1:
                 raise ConfigError('Several suitable models for {} found'.format(self.default_model_suffix))
             model = model_list[0]
+            print_info('{} - Found model: {}'.format(self.default_model_suffix, model))
         if model.suffix == '.blob':
             return model, None
         weights = network_info.get('weights', model.parent / model.name.replace('xml', 'bin'))
-
+        print_info('{} - Found weights: {}'.format(self.default_model_suffix, get_path(weights)))
```

Comment:
the same comment about moving get_path.
I mean that It should be applied when we get path in first time, not when only print it.
It logically incorrect and give mess in error logs